webapp/channels/src/components/save_button.tsx

```diff
@@ -1,10 +1,10 @@
 // Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
 // See LICENSE.txt for license information.
 
-import React from 'react';
-import {FormattedMessage} from 'react-intl';
+import React from "react";
```

Comment:
Seems like your IDE is automatically correcting some things and conflicting with our intl rules. In particular, single quotes vs double quotes.

You can fix it really simple if you use `npm run fix`, but your IDE may still be updating this for you, making future development annoying :sweat_smile: Take a look at that.

Reply:
Fixed the single quote and linting